frontend/src/screens/alby/SupportAlby.tsx

```diff
@@ -0,0 +1,294 @@
+                      </div>
+                    </li>
+                  </ul>
+                </CardContent>
+              </Card>
+              <AlertDialog open={open} onOpenChange={setOpen}>
+                <AlertDialogTrigger asChild>
+                  <Button size="lg">Become a Supporter</Button>
+                </AlertDialogTrigger>
+                <AlertDialogContent>
+                  <form onSubmit={handleSubmit}>
+                    <AlertDialogHeader>
+                      <AlertDialogTitle>Become a Supporter</AlertDialogTitle>
+                      <AlertDialogDescription>
+                        A new ZapPlanner app will be created specificially for this purpose and can be cancelled any time.
+                      </AlertDialogDescription>
+                    </AlertDialogHeader>
+
+                    <div className="flex flex-col gap-3 my-5">
+                      <div className="grid grid-cols-4 gap-4">
+                        <Label htmlFor="amount" className="text-right mt-2">
+                          Amount <br></br><span className="font-normal text-muted-foreground">(sats / month)</span>
+                        </Label>
+                        <div className="col-span-3">
+                          <Input
+                            id="amount"
+                            value={amount}
+                            onChange={(e) => setAmount(e.target.value)} />
+                          <div className="grid grid-cols-3 gap-3 mt-1">
+                            <Button type="button" variant="outline" onClick={() => setAmount("1000")}>🙏 1000</Button>
+                            <Button type="button" variant="outline" onClick={() => setAmount("5000")}>💪 5000</Button>
+                            <Button type="button" variant="outline" onClick={() => setAmount("10000")}>✨ 10000</Button>
+                          </div>
+                        </div>
+
+                      </div>
+                      <div className="grid grid-cols-4 items-center gap-4">
+                        <Label htmlFor="comment" className="text-right">
+                          Name <span className="font-normal text-muted-foreground">(optional)</span>
+                        </Label>
+                        <Input
+                          id="sender-name"
+                          value={senderName}
+                          onChange={(e) => setSenderName(e.target.value)}
+                          placeholder={`Nickname, npub, @twitter, etc.`}
+                          className="col-span-3" />
+                      </div>
+                    </div>
+
+                    <AlertDialogFooter>
+                      <AlertDialogCancel>Cancel</AlertDialogCancel>
+                      <LoadingButton
+                        type="submit"
+                        disabled={!!isSubmitting}
+                        loading={isSubmitting}
+                      >
+                        Complete Setup
```

Comment:
This feels strange to me. But I'm not sure of something better.

Reply:
Same here, was going back and forth but it's the best I could come up with for now :sweat_smile: